pyproject.toml

```diff
@@ -76,7 +77,10 @@ filterwarnings = [
     # next patch. This filter exception should also be removed once it's
     # confirmed that tests are passing again in the latest Python 3.10.
     # https://github.com/python/cpython/issues/100160
-    'default:There is no current event loop:DeprecationWarning'
+    'default:There is no current event loop:DeprecationWarning',
+
+    # https://github.com/pytest-dev/pytest-xdist/issues/825
```

Comment:
It would be great if we can add a comment here on what we can do to remove this exception similar to the other two exceptions that we added to the `filterwarnings` list. Mainly it is not clear from the referenced GitHub issue what we need to do and ideally we remove this in this exception in the future.